docs/config-file-copyright-garbage-yml.md

```diff
@@ -7,7 +7,7 @@ You can use [examples/copyright-garbage.yml](examples/copyright-garbage.yml) as
 ### When to Use
 
 No scanner is perfect and many detect code statements or binary code as copyright holders,
-`copyright-garbage.yml` provides a way to clean to clean up such errors across all scans.
+`copyright-garbage.yml` provides a way to clean up such errors across all scans.
```

Comment:
out of scope: Maybe also use the term `issues` instead of `errors` here.

Reply:
I think it's ok to keep "errors" here, because it relates to scanner errors, not our `OrtIssue` class.